addons/payment_mercado_pago/models/payment_transaction.py

```diff
@@ -170,3 +178,82 @@ def _process_notification_data(self, notification_data):
             self._set_error(
                 "Mercado Pago: " + _("Received data with invalid status: %s", payment_status)
             )
+
+    @api.model
+    def _mercado_pago_response_msg(self, verified_payment_data):
+        """ Return the response status in the human language.
+
+        :return: The response message
+        :param dict verified_payment_data: MercadoPago transaction response
+        """
+        mercadopago_messages = {
+            'accredited': _(
+                "Mercado Pago: Your payment has been credited. In your summary you will see the "
```

Comment:
Extract "Mercado Pago" from all translatable strings so that translators don't translate it to "Payment Market".